src/core_modules/capture-core/components/Pages/NewEvent/DataEntry/epics/getConvertedNewSingleEvent.js

```diff
@@ -1,9 +1,10 @@
 // @flow
-import moment from 'capture-core-utils/moment/momentResolver';
+import { moment } from 'capture-core-utils/moment';
```

Comment:
+1

Reply:
Caused by adding the default for the momentDate parameter in getFormattedStringFromMomentUsingEuropeanGlyphs function. I will keep the parameter without default so still need this here.